internal/rpcserver/interface.go

```diff
@@ -424,3 +424,43 @@ type Clock interface {
 	// Since returns the time elapsed since t.
 	Since(t time.Time) time.Duration
 }
+
+// FeeEstimator provides an interface that tracks historical data for published
+// and mined transactions in order to estimate fees to be used in new
+// transactions for confirmation within a target block window.
+//
+// The interface contract requires that all of these methods are safe for
+// concurrent access.
+type FeeEstimator interface {
+	// DumpBuckets returns the internal estimator state as a string.
+	DumpBuckets() string
+
+	// EstimateFee is the public version of estimateMedianFee. It calculates the
+	// suggested fee for a transaction to be confirmed in at most `targetConf`
```

Comment:
This specifies `targetConf` but the parameter name is actually `targetConfs` (this is the case in `Estimator.EstimateFee` as well)